app/3.0/start/toolbox/deploy.md

```diff
@@ -75,64 +68,99 @@ and create a new project.
     * Type a project name.
     * Click the Create button.
     
-    The App Engine gives you a project ID based on the name of your project.
-    Make note of this ID.
+    The App Engine gives you a unique identifier for your project-make note of this ID.
 
-1.  `cd` into the main folder for your app (e.g. `my-app/`).
-
-1. Create an `app.yaml` file with the following contents:
+1.  In the root project folder for your app (e.g. `my-app/`), create a file called `app.yaml` with the following contents:
 
     ```
     runtime: python27
     api_version: 1
-    threadsafe: yes
+    threadsafe: true
 
     handlers:
-    - url: /bower_components
-      static_dir: build/es5-bundled/bower_components
+    - url: /node_modules
+      static_dir: node_modules
       secure: always
 
     - url: /images
-      static_dir: build/es5-bundled/images
+      static_dir: images
       secure: always
 
     - url: /src
-      static_dir: build/es5-bundled/src
+      static_dir: src
       secure: always
 
     - url: /manifest.json
-      static_files: build/es5-bundled/manifest.json
-      upload: build/es5-bundled/manifest.json
+      static_files: manifest.json
+      upload: manifest.json
       secure: always
 
     - url: /service-worker.js
-      static_files: build/es5-bundled/service-worker.js
-      upload: build/es5-bundled/service-worker.js
+      static_files: service-worker.js
+      upload: service-worker.js
       secure: always
 
     - url: /.*
-      static_files: build/es5-bundled/index.html
-      upload: build/es5-bundled/index.html
+      static_files: index.html
+      upload: index.html
       secure: always
+
+    ```
+
+1.  In a text editor, open `polymer.json` from your root project folder. Add `"app.yaml"` to the `extraDependencies` array.
+
+    Before {.caption}
+
+    ```
+    ...
+    "extraDependencies": [
+        "manifest.json",
+        "node_modules/@webcomponents/webcomponentsjs/*.js",
+        "!node_modules/@webcomponents/webcomponentsjs/gulpfile.js"
+      ],
+    ...
+    ```
+
+    After {.caption}
+
+    ```
+    ...
+    "extraDependencies": [
+        "manifest.json",
+        "node_modules/@webcomponents/webcomponentsjs/*.js",
+        "!node_modules/@webcomponents/webcomponentsjs/gulpfile.js",
+        "app.yaml"
+      ],
+    ...
     ```
 
-1. Set your project id to the ID given to your app by the App Engine. For example:
+1.  Re-run `polymer build` to include `app.yaml` in your build.
+
+       polymer build
```

Comment:
I think this line and a few others below lost their code formatting because the number of spaces after the `1.` changed, causing these to no longer be 4 spaces more indented than the content after the `1.`. I'm just looking at the GitHub rendered version though, so I'm not sure if this is different when built.